jvb/src/main/kotlin/org/jitsi/videobridge/cc/OversendingTimeTracker.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.cc
+
+import org.jitsi.nlj.util.NEVER
+import java.time.Clock
+import java.time.Duration
+import java.time.Instant
+
+class OversendingTimeTracker(
```

Comment:
I started out writing this generically, but found the naming (of the class and members) awkward, as well as the API.  I figured I would do it this way and, when the second use-case comes up, perhaps it will be more obvious how it should be named/designed to handle both.